pandas/plotting/api.py

```diff
@@ -0,0 +1,20 @@
+"""
+Plotting api
+"""
+
+# flake8: noqa
+
+try:  # mpl optional
+    from pandas.plotting import converter as conv
+    conv.register()  # needs to override so set_xlim works with str/number
+except ImportError:
+    pass
+
+from pandas.plotting.misc import (scatter_matrix, radviz,
+                                  andrews_curves, bootstrap_plot,
+                                  parallel_coordinates, lag_plot,
+                                  autocorrelation_plot)
+from pandas.plotting.plotting import (boxplot, scatter_plot, grouped_hist,
```

Comment:
Not the most important :-), but the `plotting.plotting` looks a bit silly. Maybe we can call this file `base` or `core`  to make it more clear?

I am also not sure if `grouped_hist`, `hist_frame`, `hist_series` should be exposed in the public `pandas.plotting` namespace?